### DISCUSSION TITLE
Help I need a team!

### DISCUSSION THREAD

**Discussion by aruss175** [EAGE-Annual-Hackathon/EAGE-Annual-Hackathon-Discussion-Boards#4] [General]
Looking for a team to join?  Or have a really cool idea you want to hack but need help?  Ask here.

**Comment by EdwinB12** [comment]
Hi All, I'm looking for a team! 

My geophysical background is in seismic processing and to a lesser extent potential fields. I have a couple years of python and ML experience. Most of my practical experience is in training and validating convolutional neural networks.  
I'm based in Sheffield, UK and will be attending the Hackathon remotely. 

Let me know if I sound like a good match for your team! :)

**Comment by marcosjacinto** [comment]
Hello everyone!

I'm a geologist from Brazil, and I've been working as a Data Geoscientist for almost three years now using conventional ML and DL (mostly LSTM) methods. More recently, I've been focusing on and working with ML Engineering and Cloud Engineering. Moreover, I hold a Master's degree where I used a U-Net with Ground Penetration Radar data and also applied XAI to explain the predictions.

I will be attending the Hackathon in person. Looking forward to tomorrow and let me know if I can be of assistance and part of your team!
